README_CI.md

```diff
@@ -0,0 +1,109 @@
+# Jenkins CI Architecture
```

Comment:
This seems like it's more focused on just the test-running part of the CI. It doesn't talk about changelog validation or vulnerability scanning, for example. Would it be better to rename this something more specific or Is there more content to come?

Reply:
I should probably add those.  Remember i said yesterday I was only ~80% done but wanted to ship :)

Reply:
Oh gotcha, so should I just review without them for now?